gooddata-java-model/src/main/java/com/gooddata/sdk/model/util/UriHelper.java

```diff
@@ -0,0 +1,8 @@
+package com.gooddata.sdk.model.util;
+
+public abstract class UriHelper {
+
+    public static String getLastUriPart(final String uri) {
+        return uri == null ? null : uri.substring(uri.lastIndexOf("/") + 1); // TODO make safer
```

Comment:
ok, let's finish this ;)

Reply:
test and javadoc added, implementation seems robust enough